python/tank/pipelineconfig.py

```diff
@@ -741,6 +741,8 @@ def execute_core_hook_internal(self, hook_name, parent, **kwargs):
             # of the core API.
             hooks_path = os.path.abspath(os.path.join(os.path.dirname(__file__), "..", "..", "hooks"))
             hook_path = os.path.join(hooks_path, file_name)
+        else:
+            parent.log_metric("custom core hook")
```

Comment:
I think here we should include the hook name. This will allow us to answer the totango question "who has customized folder creation file permissions and I/O" for example which i think is super useful.